src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -141,14 +141,6 @@ module DotNet {
    * Receives incoming calls from .NET and dispatches them to JavaScript.
    */
   export const jsCallDispatcher = {
-    /**
-     * Finds the JavaScript function matching the specified identifier.
-     *
-     * @param identifier Identifies the globally-reachable function to be returned.
-     * @returns A Function instance.
-     */
-    findJSFunction,
```

Comment:
This was just not used. As in, there was no reason to declare it publicly since it was only used internally.